src/synopsys-action/synopsys-bridge.ts

```diff
@@ -93,18 +93,20 @@ export class SynopsysBridge {
   }
 
   async downloadBridge(tempDir: string): Promise<void> {
+    const LATEST = 'latest'
     try {
       // Automatically configure bridge if Bridge download url is provided
       let bridgeUrl = ''
       let bridgeVersion = ''
       if (inputs.BRIDGE_DOWNLOAD_URL) {
         bridgeUrl = BRIDGE_DOWNLOAD_URL
-        const versionInfo = bridgeUrl.match('.*synopsys-bridge-([0-9.]*).*')
-        if (versionInfo != null) {
-          bridgeVersion = versionInfo[1]
-        }
-        if (bridgeUrl.includes('latest')) {
-          bridgeVersion = await this.getVersionFromLatestURL()
+        if (bridgeUrl.includes(LATEST)) {
```

Comment:
can we check latest as case insensitively?

Reply:
I have referred this url its specified it as "latest" so using it.
https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/synopsys/integration/synopsys-bridge/latest/
https://artifactory.internal.synopsys.com/artifactory/clops-local/clops.sig.synopsys.com/synopsys-bridge/latest/
https://artifactory.internal.synopsys.com/artifactory/clops-local/clops.sig.synopsys.com/bridge/latest/